test/common/secret/sds_api_test.cc

```diff
@@ -14,6 +14,7 @@
 
 #include "test/common/stats/stat_test_utility.h"
 #include "test/mocks/config/mocks.h"
+#include "test/mocks/event/mocks.h"
```

Comment:
Why is this needed in the revert? Seems unrelated?

Reply:
In my code I added it to mocks/grpc/mocks.h,so it was includes transitivly.. after the revert it is no longer in grpc.h